### PR TITLE
Use structopt crate instead of clap.

### PR DESCRIPTION
**Pull request by gevorgyana**
Signed-off-by: Artyom Gevorgyan <artemhevorhian@gmail.com>

### CODE REVIEW

**Comment by gevorgyana** [comment]
closes #145

File: tools/layoutflash/Cargo.toml [hunk 1]
```diff
@@ -8,4 +8,4 @@ edition = "2018"
 device_tree = { path = "../../src/lib/device_tree" }
 model = { path = "../../src/drivers/model" }
 wrappers = { path = "../../src/drivers/wrappers" }
-clap = { git = "https://github.com/clap-rs/clap" }
+structopt = { git = "https://github.com/TeXitoi/structopt" }
```
Comment (doriath) [inline tools/layoutflash/Cargo.toml:11]: Can we just specify the version instead of using git path:

```toml
structopt = "0.3"
```

**Comment by doriath** [comment]
Can we reference #145?

**Review by rminnich — COMMENTED** [review]
it would be good to have more of an explanation. Why the change? What improvement did it bring?

**Comment by gevorgyana** [comment]
Should probably ask the author of #145 to explain why he opened the issue or wait/close this otherwise. To me, this change is not necessary.

**Comment by rjoleary** [comment]
It looks like the author of #145 closed it, so closing this too.
Let me know if I misunderstood. If so, we can easily re-open.

**Comment by gevorgyana** [comment]
Is it okay that we merged the changes from https://github.com/oreboot/oreboot/pull/279 but closed the issue? I mean, the PR is now marked as red on github, but the code is merged. Or it doesn't matter? This is a toy PR, but I am just learning the workflow.

**Comment by fotonick** [comment]
@gevorgyana It's just on this PR-associated set of branches. It's [not on real master](https://github.com/oreboot/oreboot/blob/master/tools/layoutflash/Cargo.toml). Also clicking on that commit, I see, "This commit does not belong to any branch on this repository".

**Comment by gevorgyana** [comment]
Some context as I removed my previous message. I was confused by the fact that this PR is closed, and at the same time [I thought!] this commit [this commit](https://github.com/oreboot/oreboot/pull/279/commits/4b2417d009307203f970839e1cde87da546d4660)'s purpose was to merge my changes into the oreboot/master. While in reality it was to "update" my branch with the changes from the real master, same thing if I ran git pull with fetch path being oreboot/master. Seems like it is frequently done by maintainers on various PR, for example the same message can be seen [here](https://github.com/oreboot/oreboot/pull/290)

P.S. I removed the fork, therefore we can no longer see the changes.

**Pull request closed without merging**